app/src/main/java/at/roteskreuz/stopcorona/screens/reporting/ReportingActivity.kt

```diff
@@ -27,17 +28,29 @@ class ReportingActivity : CoronaPortraitBaseActivity() {
 
     companion object {
         private const val ARGUMENT_MESSAGE_TYPE = "argument_message_type"
+        private const val ARGUMENT_DATE_WITH_MISSING_EXPOSURE_KEYS =
+            "argument_upload_missing_exposure_keys"
 
-        fun args(messageType: MessageType): Bundle {
+        fun args(messageType: MessageType, dateWithMissingExposureKeys: ZonedDateTime?): Bundle {
             return bundleOf(
-                ARGUMENT_MESSAGE_TYPE to messageType
+                ARGUMENT_MESSAGE_TYPE to messageType,
+                ARGUMENT_DATE_WITH_MISSING_EXPOSURE_KEYS to dateWithMissingExposureKeys
```

Comment:
I guess there is an error, because ZonedDateTime is not parcalable. 
You need to parcel it manually (to the string for example) and then restore it from the parceled value to the `dateWithMissingExposureKeys` variable.

Reply:
updated